test/widgets/test_nvidia_sensors.py

```diff
@@ -12,3 +15,44 @@ def test_nvidia_sensors_input_regex():
     assert correct_sensors == {'temp', 'fan_speed', 'perf'}
     assert _all_sensors_names_correct(correct_sensors)
     assert not _all_sensors_names_correct(incorrect_sensors)
+
+
+def no_op(*args, **kwargs):
+    print("no_op")
```

Comment:
You can delete this line. Appreciate you copied it from one of the tests I wrote but it's not needed!